Mariuzzo.DO.DataAnnotations/CedulaAttribute.cs

```diff
@@ -13,7 +13,10 @@ public sealed class CedulaAttribute : ValidationAttribute
     {
         
         private readonly IValidator _cedulaValidator = new CedulaValidator();
-
+        /// <summary>
+        /// Verify some exceptions of valid Cédula that don't follow the Luhn Algorithm
+        /// </summary>
+        public bool IgnoreExceptions = true;
```

Comment:
This isn't really needed. If the standard for the Dominican cédula determines that there are exceptions, and there's a specific list for it, those exceptions should be valid at all times.